models/articles.js

```diff
@@ -11,7 +29,7 @@ function selectArticles() {
           JOIN comments
           on articles.article_id = comments.article_id 
           GROUP BY articles.article_id
-          ORDER BY articles.created_at DESC`
+          ORDER BY articles.${sort_by} ${order.toUpperCase()}`
```

Comment:
This leaves your server open to SQL injection. What can you use instead of string interpolation to create this query ?

Reply:
Hi Jo, I thought we couldn't use parameterized queries ($1) with ORDER BY statements and green listing would be sufficient although could use pg format to sanitise the data.